plugins/blockchain_statistics/include/golos/plugins/blockchain_statistics/runtime_bucket_object.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include <appbase/application.hpp>
+#include <golos/plugins/chain/plugin.hpp>
+#include <golos/plugins/blockchain_statistics/bucket_object.hpp>
+#include <golos/chain/steem_object_types.hpp>
+
+
+namespace golos {
+namespace plugins {
+namespace blockchain_statistics {
+
+using namespace golos::chain;
+
+struct runtime_bucket_object {
+
+    std::vector < std::string > get_as_string ();
```

Comment:
move to cpp

Reply:
What .cpp file do you mean? I don't understand at all

Reply:
Alexander asked you to move this structure to plugin.cpp.

I think, you may keep runtime_bucket_object in separated header.